protocol/communities_messenger_token_permissions_test.go

```diff
@@ -1736,7 +1768,7 @@ func (s *MessengerCommunitiesTokenPermissionsSuite) TestResendEncryptionKeyOnBac
 
 	waitOnChannelKeyToBeDistributedToBob := s.waitOnKeyDistribution(func(sub *CommunityAndKeyActions) bool {
 		action, ok := sub.keyActions.ChannelKeysActions[chat.CommunityChatID()]
-		if !ok || action.ActionType != communities.EncryptionKeyAdd {
+		if !ok || action.ActionType != communities.EncryptionKeySendToMembers {
```

Comment:
can't really tell why this side-effect. But The members are sent on `EncryptionKeySendToMembers`, not on `EncryptionKeyAdd`